files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -694,9 +694,23 @@ public function mkdir( $path, $mode, $options ) {
 	 * @return  bool
 	 */
 	public function stream_metadata( $path, $option, $value ) {
-		$this->debug( sprintf( 'stream_metadata =>  %s + %s + %s', $path, $option, $value ) );
+		$this->debug( sprintf( 'stream_metadata =>  %s + %s + %s', $path, $option, json_encode( $value ) ) );   // phpcs:ignore WordPress.WP.AlternativeFunctions.json_encode_json_encode
```

Comment:
`$value` can be an array; `json_encode()` is used to avoid "array to string conversion" warnings.